Kubernetes/omsagent.yaml

```diff
@@ -398,7 +400,7 @@ spec:
          memory: 500Mi
         requests:
          cpu: 50m
-         memory: 100Mi
+         memory: 175Mi
```

Comment:
revert to 100Mi?

Reply:
i am still not sure if we should ship rs with fb & telegraf enabled for this release. But we definetely need to for next release. If we remove, i will revert. Also, leaving the requests raised, does not hurt much.